### PR TITLE
Remove MU from the blueprint

### PR DESCRIPTION
**Pull request by zinyando**
Since MU has been removed from Octane scope I think we might also need to remove it from the blueprints.

https://blog.emberjs.com/2019/03/11/update-on-module-unification-and-octane.html

### CODE REVIEW

File: packages/@ember/octane-addon-blueprint/files/tests/dummy/config/environment.js [hunk 1]
```diff
@@ -4,7 +4,6 @@ module.exports = function(environment) {
   let ENV = {
     'ember-resolver': {
       features: {
-        EMBER_RESOLVER_MODULE_UNIFICATION: true
```
Comment (rwjblue) [inline packages/@ember/octane-addon-blueprint/files/tests/dummy/config/environment.js]: The whole `'ember-resolver'` section can be removed, mind updating that?

File: packages/@ember/octane-app-blueprint/files/config/environment.js [hunk 2]
```diff
@@ -4,7 +4,6 @@ module.exports = function(environment) {
   let ENV = {
     'ember-resolver': {
       features: {
-        EMBER_RESOLVER_MODULE_UNIFICATION: true
```
Comment (rwjblue) [inline packages/@ember/octane-app-blueprint/files/config/environment.js]: ^ RE: `'ember-resolver'` section

**Comment by zinyando** [comment]
@rwjblue In the `octane-app-blueprint` app folder do I need to add the other files as well that are missing but are there in the current 'classic' blueprints eg
- controllers
- helpers
- templates
- models

**Comment by rwjblue** [comment]
@zinyando I don't think they are "required", but its fine to just mirror what `ember-new-output` has inside `app/`.  
https://github.com/ember-cli/ember-new-output

**Pull request merged**